### PR TITLE
Set separate jshint standards for the node app and the polyfills library

### PR DESCRIPTION
**Pull request by triblondon**
I haven't had jshint turned on up to now, and I wanted to fix that, but I realised that the current jshintrc config targets polyfills, not server code.  So I created a second jshintrc in the polyfills directory to target polyfill code, and modified the original to be more appropriate to the node app.

The settings for the app are FT standard, and I've pretty much kept Jonathan's suggested settings for the polyfills.

### CODE REVIEW

**Comment by triblondon** [comment]
@samgiles could you review?  Most important is the two jshintrc files, the rest is updating the service code to conform to the new standard.

**Comment by triblondon** [comment]
@jonathantneal said on chat this evening that this has his OK.

**Comment by jonathantneal** [comment]
Yes, it does. I especially love the `globalstrict`.

File: .jshintrc [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+	"unused": "vars",
 	"boss": true,
+	"esnext": true,
+	"expr": true,
+	"sub": true,
+	"node": true,
 	"strict": false,
-	"predef": ["Window", "HTMLDocument", "HTMLElement"]
+	"globalstrict": true,
+	"curly": false
```
Comment (samgiles) [inline .jshintrc]: This doesn't seem to be in the spec: http://origami.ft.com/docs/syntax/js/#syntax-convention-rules

Disclaimer: I'm anti-single line branching statements.

**Comment by samgiles** [comment]
~I only see one `.jshintrc`?~

Ignore me

**Comment by samgiles** [comment]
Bar inline comment, LGTM.

**Pull request merged**